src/js/cookie-policy.js

```diff
@@ -12,9 +12,20 @@ if (ubuntu.hasOwnProperty('cookiePolicy')) {
 // The cookie policy injection and interaction
 ubuntu.cookiePolicy = function() {
   let context = null;
+  let options = {
+    'content':
+      `We use cookies to improve your experience. By your continued
+      use of this site you accept such use. To change your settings
+      please
+      <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">
+        see our policy
+      </a>.`
+  }
 
   return {
-    setup: function(content) {
+    setup: function(options) {
+      let content = options.content;
+      let duration = options.duration;
       let start = `
         <div class="p-notification--cookie-policy">
```

Comment:
As this is a popup and something that can be closed or hidden, I feel this should be using the ```<dialog>``` html element. Also add ```aria-labelledby``` or ```aria-describedby``` to give it more context and improve the accessibility.